spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'User Show Page', type: :feature do
+  before do
+    # Set up any necessary data or visit relevant pages
+  end
+
+  it 'displays the user profile picture' do
+    # Test code goes here
+  end
+
+  it 'displays the user username' do
+    # Test code goes here
+  end
+
+  it 'displays the number of posts the user has written' do
+    # Test code goes here
+  end
+
+  it 'displays the user bio' do
+    # Test code goes here
+  end
+
+  it 'displays the first 3 posts of the user' do
+    # Test code goes here
+  end
+
+  it 'redirects to post show page when clicking on a user post' do
+    # Test code goes here
+  end
+
+  it 'redirects to user post index page when clicking to see all posts' do
+    # Test code goes here
+  end
+end
```

Comment:
- [ ] Good to have this user show test file. :+1:  But please from the project requirement, you should write a test for the following :point_down: But now you have no test code in your `it` block to test the following:blush: 
  
  * Tests that asserts you can see the user's profile picture.
  * Tests that asserts you can see the user's username.
  * Tests that asserts you can see the number of posts the user has written.
  * Tests that asserts you can see the user's bio.
  * Tests that asserts you can see the user's first 3 posts.
  * Tests that asserts you can see a button that lets me view all of a user's posts.
  * Tests that redirects to the post's show page when you click on a user's post.
  * Tests that redirects to the user's post's index page when you click on a **see all posts**.